### PR TITLE
Add contributing guidelines

### PR DESCRIPTION
**Pull request by dwisiswant0**
To maintain an ecosystem of nuclei templates, we should have contributing guidelines. Next will be done by the PD team.

- [x] PR template
- [x] Code of Conduct 
- [x] Contributing guidelines

cc: @bauthard

### CODE REVIEW

**Comment by bauthard** [comment]
Thank you for proposing this idea, we will work on this.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+<!-- Demonstrate the template is solid. Example: The exact commands you ran template and their output or/ screenshots. -->
+<!-- NOTE: REDACTED VULNERABLE TARGET (if any)! -->
+
+### Checklists:
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+
+- [ ] I have followed the guidelines in our [CONTRIBUTING.md](https://github.com/projectdiscovery/nuclei-templates/blob/master/.github/CONTRIBUTING.md) document.
```
Comment (toufik-airane) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: The link is down. The good one seems to be 
https://github.com/projectdiscovery/nuclei-templates/blob/master/CONTRIBUTING.md.

**Review by toufik-airane — CHANGES_REQUESTED** [review]
Interesting initiative. 👍 

We should inspire from the [contribution guideline](https://github.com/projectdiscovery/nuclei-templates/blob/master/.github/CONTRIBUTING.md).

**Comment by dwisiswant0** [comment]
My remote branch is gone. Move to #2236.

**Pull request closed without merging**